reference/api/search.md

```diff
@@ -385,6 +385,8 @@ You can use `hitsPerPage` and `page` to [paginate search results](/learn/advance
 
 ::: note
 `hitsPerPage` and `page` take precedence over `offset` and `limit`. If a query contains either `hitsPerPage` or `page`, any values passed to `offset` and `limit` are ignored.
+
+`hitsPerPage` and `page` are resource-intensive options and might negatively impact search performance. This is particularly likely if [`maxTotalHits`](/reference/api/settings.md) is set to a value higher than its default.
```

Comment:
Can't we physically highlight this, in a warning (instead of notes) or with ⚠️?

There is a lot of text on this page, I would like their eyes to catch what you just added